client/src/Components/styles/TopPanel.css

```diff
@@ -1,6 +1,7 @@
 #top-panel {
     border: 1px solid black;
-    background-image: url("https://i.pinimg.com/originals/39/2f/38/392f38884a695299b0b0a295f5edb1a2.png");
+    background-image: url(../../cartoon-background.jpg);
```

Comment:
Should this be `cartoon-background2.png`? or have you verified this works?
Also can you rename `cartoon-background2.png` to `cartoon-background.png`? thanks!

Reply:
it works, cartoon background 2 is the old one